static/styles/deckviewer_styles.css

```diff
@@ -32,13 +32,103 @@ body.frogtown {
   background-image: url("https://www.frogtown.me/Images/CardBack.jpg");
 }
 
-#root{
+#root {
   display: flex;
   flex-direction: row;
   height: calc(100% - 64px);
 }
 
-#deckArea{
+#altPane {
+  visibility: hidden;
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+}
+
+#altContainer {
+  height: calc(100% - 130px);
+  margin: 35px;
+  margin-top: 100px;
+  position: fixed;
```

Comment:
Only `#altPane` should have `position: fixed`. This and `#altDim` should have `position: absolute;`. It all works as expected, but it's more clear what your intention is.